### PR TITLE
Add deliverable page, overall css, and more

### PR DESCRIPTION
**Pull request by RadioPotin**
Fixes: #18 #16 #15 #17

- added `deliverables / publications` page
- add css considerations, partial responsiveness
- various fixes like typos, missing logos, header adjustments

### CODE REVIEW

**Review by MuSSF — APPROVED** [review]
Bravo, mille fois mieux que l'ancienne version !

**Review by MuSSF — APPROVED** [review]

File: content/fr/deliverables.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: "Livrables"
+synopsis: "Le projet DéCySif fourni un cadre pour que le consortium puisse aboutir à un ensemble de livrables ainsi qu'une suite de papiers scientifiques divers qui seront réunis ci-dessous à mesure qu'ils voient le jour."
+project_deliverables:
+  repository: "https://github.com/Decysif/livrables/tree/master"
+  lots: 
+    - lot_nb: "Lot 1"
+      papers:
+        - id: "L1.1"
+          name: "Constitution d’une base de fichiers d’entrée représentatifs des difficultés rencontrées pour générer des exploits."
+          synopsys: "Bientôt"
```
Comment (MuSSF) [inline content/fr/deliverables.md:13]:
```suggestion
          synopsys: "Ce livrable est constitué d’une base de tests qui se trouve dans le dépôt ’benchmarks’ du projet
Décysif. Les objectifs du livrable sont de repérer les faiblesses du prouveur Alt-Ergo, les problèmes de traduction (ou repérer des problèmes au niveau de l’écriture des théories, par exemple le modèle mémoire de J3) pour tous les prouveurs cvc5, CVC4, Z3, Alt-Ergo, et d'identifier des faiblesses lors de la reconstruction d’un contre-exemple par Why3 depuis les modèles des solveurs SMT, ou les faiblesses des procédures de vérification et de catégorisation des contre-exemples."
```

File: content/fr/deliverables.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          leader: "TrustInSoft"
+          authors:
+            - "Guillaume Cluzel (TrustInSoft)"
+            - "Matteo Manighetti (Inria & Université Paris-Saclay)"
+            - "Claude Marché (Inria & Université Paris-Saclay)"
+            - "Yannick Moy (AdaCore)"
+          pdf_url: "/pdf/livrable_l1_1.pdf"
+        - id: "L2.1"
+          name: "Constitution d’une base de fichiers d’entrée représentatifs des difficultés rencontrées pour la preuve automatique."
+          synopsys: "Bientôt"
```
Comment (MuSSF) [inline content/fr/deliverables.md:23]:
```suggestion
          synopsys: "Ce livrable est constitué d’une base de tests qui se trouve dans le dépôt ’benchmarks’ du projet
Décysif, dont les objectifs sont de repérer les faiblesses du prouveur Alt-Ergo, de repérer les problèmes de traduction (ou repérer des problèmes au niveau de l’écriture des théories, par exemple le modèle mémoire de J3) pour tous les prouveurs CVC5, CVC4, Z3, Alt-Ergo.
```

**Review by Ninjapouet — APPROVED** [review]

**Comment by RadioPotin** [comment]
This merge broke the build job on master because the YAML was incorrectly formatted, working on a fix rn, adding english version of the suggested content at the same time too

**Pull request merged**